node/src/service.rs

```diff
@@ -89,7 +90,7 @@ type FullBackend = TFullBackend<Block>;
 pub struct ParachainNativeExecutor;
 
 impl sc_executor::NativeExecutionDispatch for ParachainNativeExecutor {
-    type ExtendHostFunctions = frame_benchmarking::benchmarking::HostFunctions;
+    type ExtendHostFunctions = ParachainHostFunctions;
```

Comment:
@girazoki do benchmarks work? Maybe this breaks it by removing the benchmarking host functions

Reply:
I have not tried, maybe we have to put a tuple here?

Reply:
Fixed, benchmarks working again